apps/web/src/components/display/UserManagementTable.tsx

```diff
@@ -0,0 +1,100 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import sortIcon from '@assets/img/sort.svg';
+import { buttonBase, buttonColor } from '@components';
+import { EmployeeRO, formatDate } from '@ien/common';
+import { Spinner } from '../Spinner';
+
+export interface UserManagementProps {
+  employees: EmployeeRO[];
+  loading: boolean;
+  onSortChange: (field: string) => void;
+}
+
+export const UserManagementTable = (props: UserManagementProps) => {
+  const { employees, loading, onSortChange } = props;
+  const router = useRouter();
+
+  return (
+    <div className='overflow-x-auto'>
+      <table className='text-left w-full'>
+        <thead className='whitespace-nowrap bg-bcLightGray text-bcDeepBlack'>
+          <tr className='border-b-2 border-yellow-300 text-sm'>
+            <th className='px-6 py-4'>
+              <div className='flex align-middle justify-between'>
+                <span>Name</span>
+                <button id='sort-by-name' onClick={() => onSortChange('name')}>
+                  <img src={sortIcon.src} alt='sort' />
+                </button>
+              </div>
+            </th>
+            <th className='px-6'>Email Address</th>
+            <th className='px-6'>
+              <div className='flex align-middle justify-between'>
+                <span>Created On</span>
+                <button id='sort-by-name' onClick={() => onSortChange('name')}>
```

Comment:
id and argument of onSortChange are wrong. btw, why don't you use SortButton?

Reply:
correct, that component was not created when I did this part, will change to sort button comp